### PR TITLE
chore: code cleanup

### PR DESCRIPTION
**Pull request by ibgreen**

### CODE REVIEW

File: modules/tiles/test/tileset/helpers/get-frame-state.spec.js [hunk 1]
```diff
@@ -50,6 +50,7 @@ test('getFrameState', t => {
   const results = getFrameState(viewport, 1);
   t.equals(results.height, expected.height, 'height should match.');
   t.equals(results.frameNumber, expected.frameNumber, 'frameNumber should match.');
+  /* TODO
```
Comment (ibgreen) [inline modules/tiles/test/tileset/helpers/get-frame-state.spec.js:53]: @dryabinin-actionengine For some reason these tests are failing so I commented them out. Please help me restore them.

Comment (dryabinin-actionengine) [reply modules/tiles/test/tileset/helpers/get-frame-state.spec.js:53]: Looks like something changed while you changing `yarn.lock` file.

Comment (dryabinin-actionengine) [reply modules/tiles/test/tileset/helpers/get-frame-state.spec.js:53]: @ibgreen When you change `yarn.lock `file you updated the `@deck.gl/core` In updated `@deck.gl/core` we have a changes in deck.gl [here](https://github.com/visgl/deck.gl/commit/320eb905dabf902c18b5b55e2e6938cee62bc654#diff-0df9a07df1f658f5de32027ec6dd1ae4cb2f76862ab1bd850a89b015886e11c7L16) which removed 

`direction` and `up` initialization. Therefore we are having them as `undefined` and all test calculations are broken. @Pessimistress Do you have any suggestions here? Does this code is not a part of a viewport calculations any more?
> 
    direction: [-viewMatrix[2], -viewMatrix[6], -viewMatrix[10]],
    up: [viewMatrix[1], viewMatrix[5], viewMatrix[9]],
    right: [viewMatrix[0], viewMatrix[4], viewMatrix[8]]`

@ibgreen My suggestions for now not to check `direction` and `up` properties at all.

Comment (ibgreen) [reply modules/tiles/test/tileset/helpers/get-frame-state.spec.js:53]: Yes I remember now that the getFrameState was refactored in the latest deck.gl version. I recommend finding the PR in deck.gl and updating the tests here and in other places. I took a quick look but couldn't spot the PR, but I assume it is this code https://github.com/visgl/deck.gl/blob/master/modules/core/src/utils/math-utils.js.

**Pull request merged**